src/components/bottom-bar/_bottom-bar.scss

```diff
@@ -1,3 +1,25 @@
+@mixin bottom-bar-gradient {
```

Comment:
just sharing my thoughts:
1. if i understand correctly, `bottom-bar-gradient` is the same gradient being used both in `bottom-bar` and `top-bar`. if this is the case, should we give it a different name and not limit it to `bottom-bar`?
2. also, if we're using it in both places, should we declare the `mixin` in a general place, rather than in `bottom-bar`?